benchmark.py

```diff
@@ -300,6 +348,30 @@ def main():
                       default="0")
   parser.add_argument("--database", help="name of the database",
                       default="envoy_stat_db")
+  parser.add_argument("--arrangement", help=("the type of arrangement in"
+                                             " this experiment."),
+                      default="single-vm-permanent")
+  parser.add_argument("--nginx_cores",
+                      help="the start and end core numbers for Nginx server to "
+                           "run, separated by a comma.",
+                      default="0,10")
+  parser.add_argument("--envoy_cores",
+                      help="the start and end core numbers for"
+                           " Envoy to run, separated by a comma.",
+                      default="11,18")
+  parser.add_argument("--h2load_cores",
+                      help="the start and end core numbers for "
+                           "h2load to run, separated by a comma.",
+                      default="19,19")
```

Comment:
The default seems somewhat imbalanced. Having a single core for client load generation seems likely to bottleneck at the client. Can we adjust these defaults based on your current findings?